project/Dependencies.scala

```diff
@@ -14,8 +14,10 @@ object Dependencies {
   }
 
   // changing the Scala version of the Java SDK affects end users
-  val ScalaVersion = "2.13.14"
+  val CrossScalaVersions = Seq("2.13.14", "3.3.3")
+  val ScalaVersion = CrossScalaVersions.head
   val ScalaVersionForTooling = "2.12.19"
+  val Scala3Version = "3.3.3"
```

Comment:
```suggestion
```
Can be removed if we use `CrossScalaVersions`

Reply:
Not sure I fully understood your suggestion.. you would rather have the version directly on the `CrossScalaVersions` val? see what you think of current setup anyway.

Reply:
Looks good now. 

Either

```scala
val ScalaVersion = "2.13.14"
val Scala3Version = "3.3.3"
val CrossScalaVersions = Seq(ScalaVersion, Scala3Version)
```

or

```scala
val ScalaVersion = CrossScalaVersions.head
val CrossScalaVersions = Seq("2.13.14", "3.3.3")
```

I don't remember how it was before, but looks good now.